### PR TITLE
Add check to description

### PR DESCRIPTION
**Pull request by chrispouliot**
Check if ID exists in PR description

### CODE REVIEW

**Comment by chrispouliot** [comment]
Tests are passing!

File: src/github.js [hunk 1]
```diff
@@ -7,8 +7,13 @@ const log = require('./log')('agpw:github')
...
 module.exports.getAsanaId = function (data) {
-  var toMatch = data['pull_request']['title']
-  return match(toMatch)
+  var title = data['pull_request']['title']
+  var body = data['pull_request']['body']
+  var m = match(title)
+  if (!m) {
+    m = match(body)
+  }
+  return m
```
Comment (codeallthethingz) [inline src/github.js]: My JS is weak, but could this be

`return match(title) | match(body)`

File: src/github.js [hunk 2]
```diff
@@ -29,17 +34,21 @@ module.exports.shouldProcess = function (data) {
   }
 
   var title = data.pull_request.title
+  var body = data.pull_request.body
   var change = data.changes && data.changes.title && data.changes.title.from ? data.changes.title.from : null
-  var titleId = match(title)
+  var id = match(title)
+  if (!id) {
+    id = match(body)
+  }
```
Comment (codeallthethingz) [inline src/github.js]: Lines 36 - 42 (excluding 38) can be replaced with `module.exports.getAsanaId`

Comment (codeallthethingz) [reply src/github.js]: or `exports.getAsanaId` maybe?

Comment (chrispouliot) [reply src/github.js]: done

**Pull request merged**